scripts/configure.py

```diff
@@ -122,8 +122,21 @@
     'apt' : [ 'pianobar']
   },
   'airplay' : {
-    'apt' : [ 'shairport-sync' ],
-    'copy' : [{'from': 'bin/ARCH/shairport-sync-metadata-reader', 'to': 'streams/shairport-sync-metadata-reader'}],
+    'apt' : [ 'shairport-sync'],
+    'copy' : [{'from': 'bin/ARCH/shairport-sync-ap2', 'to': 'streams/shairport-sync-ap2'},
+              {'from': 'bin/ARCH/shairport-sync', 'to': 'streams/shairport-sync'}],
+    'script': [
+        'if which nqptp  > /dev/null; then exit 0; fi',
+        'cd ~',
+        'git clone https://github.com/mikebrady/nqptp.git',
```

Comment:
Suggestion: use a temporary directory

Reply:
take a look at what I changed it to, not sure if what I did works or now